privacyidea/lib/tokens/yubicotoken.py

```diff
@@ -138,6 +138,7 @@ def check_otp(self, anOtpVal, counter=None, window=None, options=None):
 
         apiId = get_from_config("yubico.id", DEFAULT_CLIENT_ID)
         apiKey = get_from_config("yubico.secret", DEFAULT_API_KEY)
+        YUBICO_URL = get_from_config("yubico.url", DEFAULT_YUBICO_URL)
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -138,6 +138,7 @@ def check_otp(self, anOtpVal, counter=None, window=None, options=None):
>>  
>>          apiId = get_from_config("yubico.id", DEFAULT_CLIENT_ID)
>>          apiKey = get_from_config("yubico.secret", DEFAULT_API_KEY)
>> +        YUBICO_URL = get_from_config("yubico.url", DEFAULT_YUBICO_URL)
>
> Yes. Further below in the exception branch is also an occurance of YUBICO_URL -> yubico_url

Hopefully done.
Please see https://github.com/privacyidea/privacyidea/pull/354/commits

I'll be unavailable in the next few days...

-- 
The only problem with troubleshooting is that the trouble shoots back.